feature

sangyun5108 opened the pull request.

sangyun5108 committed:
docs header

Make two header types

sangyun5108 committed:
Make lecture evaluation table & accommodate codereview

sangyun5108 committed:
Move file

SimYunSup commented on bcsd_hangang/src/components/MainBigPage.js:
```diff
@@ -0,0 +1,45 @@
+    height:75px;
+    padding-top:200px;
+`;
+const GoHangangContextQuestion = styled.div`
```
`h1` 태그로 바꿔주세요.

SimYunSup commented on bcsd_hangang/src/components/MainBigPage.js:
```diff
@@ -0,0 +1,45 @@
+import MainPageLectureRanking from './MainPageLectureRanking';
+import styled from 'styled-components';
+
+const GoHangangImgAndContextWrapper = styled.div`
```
변수명은 `MainPageBanner`같은 식으로 지어주세요.

SimYunSup commented on bcsd_hangang/src/components/MainPage.js:
```diff
@@ -16,7 +16,7 @@ const MainPage = () => {
     return (
     <>
         {window.addEventListener('resize',checkScreenWidth)}
```
useEffect를 사용하세요.

SimYunSup commented on bcsd_hangang/src/components/MainBigPage.js:
```diff
@@ -0,0 +1,45 @@
+const MainBigPage = () => {
+    return(
+        <>
+            <MainPageBigHeader/>
```
확장성을 위해 `Header`가 밖에 있어야 합니다.

SimYunSup commented on bcsd_hangang/src/customHooks/useGetLectures.js:
```diff
@@ -0,0 +1,37 @@
+import {
```
고치세요

SimYunSup reviewed (changes requested):
바꿔주세요 :)

sangyun5108 committed:
Make lists of department

sangyun5108 committed:
Acommodate Code Reveiw in MainPage.js

이벤트 처리 함수 등록 및 해제를 useEffect를 사용하여 선언해주었습니다.

sangyun5108 committed:
Accommodate codereview except redux-toolkit-query

sangyun5108 committed:
Make section components located at the center of mainpage

sangyun5108 committed:
Exchange margin px to auto

sangyun5108 committed:
Adjust space of tags in all page

sangyun5108 committed:
Apply font-family

sangyun5108 committed:
Make a difference in LectureRankingTitle's color when it was clicked

sangyun5108 committed:
Delete contentbox's width

sangyun5108 committed:
Make footer in big page

sangyun5108 committed:
Make footer in MainSmallPage.js

sangyun5108 committed:
Change color that is used light-grey

sangyun5108 committed:
Change default value which shows correct responsive screen of MainPage.js

sangyun5108 committed:
Edit MainPageBigHeader.js

sangyun5108 committed:
Make lists of timetable

sangyun5108 committed:
Make lists in MainPageRecentLectures.js

sangyun5108 committed:
Reduce amount of code in endpoints

sangyun5108 committed:
Check errors

sangyun5108 committed:
Make file named pages

sangyun5108 committed:
Edit header part of MainPageBig

sangyun5108 committed:
Edit file path './Store/store' to './Redux/store in index.js

sangyun5108 committed:
Delete min-width for responsive web

sangyun5108 committed:
Edit ranking of lecture table for responsive web

sangyun5108 committed:
Edit percent of width that is related table

sangyun5108 committed:
Edit list of department table for responsive web

sangyun5108 committed:
Edit timetable for responsive web

To satisfy screen of many different phones, edit timetable for responsive web and  font-size

sangyun5108 committed:
Delete gab between department and timetable

sangyun5108 committed:
Edit materials of lecture for responsive web

To satisfy screen of many different phones, edit materials of lecture for responsive web

sangyun5108 committed:
Revise order of list

Revise order of list "01" to "1"

sangyun5108 committed:
Edit table of lectures for responsive web

To satisfy screen of many different phones, edit table of lectures that recently showed  for responsive web

sangyun5108 committed:
Delete border-bottom of last list

Delete border-bottom of last list because  the bottom of outer border has two lines

sangyun5108 committed:
Change boder weight of all table

Change boder weight of all table that shows 2px to 1px

sangyun5108 committed:
Solve error of localStorage

When delete localstorage, error that means list of null occurred. So, edit code to check localstorage whether exist

sangyun5108 committed:
Change boder color

Change border color to rgb(238,238,238)

sangyun5108 committed:
Edit position of title

Edit position of table title to naturally show in big page
